app/kubernetes-ingress-controller/version-compatibility.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Version Compatibility
+
+description: |
+  Which versions of {{ site.kic_product_name }} are compatible with specific versions of {{ site.base_gateway }}, Kubernetes, Gateway API and Istio?
+
+content_type: reference
+layout: reference
+
+products:
+  - kic
+
+works_on:
+  - on-prem
+  - konnect
+---
+
+
+{{ site.kic_product_name }} is compatible with different flavors of {{ site.base_gateway }}.
+
+## {{ site.base_gateway }}
+
+{% version_compatibility_table %}
+product: "{{ site.kic_product_name }}"
+versions: 
+  - 2.12
+  - 3.0
+  - 3.1
+  - 3.2
+  - 3.3
+  - 3.4
+compatible_product: "{{ site.base_gateway }}"
+compatible_versions:
+  2.8.x: [2.12]
+  3.4.x: [2.12, 3.0, 3.1, 3.2, 3.3, 3.4]
+  3.5.x: [2.12, 3.0, 3.1, 3.2, 3.3, 3.4]
+  3.6.x: [2.12, 3.0, 3.1, 3.2, 3.3, 3.4]
+  3.7.x: [2.12, 3.0, 3.1, 3.2, 3.3, 3.4]
+  3.8.x: [2.12, 3.0, 3.1, 3.2, 3.3, 3.4]
+  3.9.x: [2.12, 3.0, 3.1, 3.2, 3.3, 3.4]
+  3.10.x: [2.12, 3.0, 3.1, 3.2, 3.3, 3.4]
+{% endversion_compatibility_table %}
+
+## Kubernetes
+
+### General
+
+The following table presents the general compatibility of {{site.kic_product_name}} with specific Kubernetes versions.
+Users should expect all the combinations marked with <i class="fa fa-check"></i> to work and to be supported.
```

Comment:
```suggestion
```
The icon isn't rendering correctly here, and the icons in the table make it easy enough to understand what is and isn't supported, so this can just be removed.